rules/cross-platform/credential_access_forced_authentication.toml

```diff
@@ -0,0 +1,65 @@
+[metadata]
+creation_date = "2024/07/22"
+integration = ["endpoint", "system"]
+maturity = "production"
+updated_date = "2024/07/22"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a potential forced authentication. Attackers may attempt to force targets to authenticate to a Linux machine
+controlled by them to capture hashes or enable relay attacks.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.network-*", "logs-system.security-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Active Directory Forced Authentication from Linux Host"
```

Comment:
```suggestion
name = "SMB Authentication from a Linux Host"
```

Reply:
Not sure if this explains better what it tries to detect

Reply:
forced auth means we have evidence that the DC is connecting back to the linux box, so maybe inverting the sequence (auth from window followed by incoming netcon to linux) ?

Reply:
++